includes/class_award_force_api.inc

```diff
@@ -101,25 +61,25 @@ class AwardForceAPI
         return new Client([
             'base_uri' => $this->apiUrl,
             'headers' => [
-                'Accept' => 'application/vnd.Award Force.v1.0+json',
-                'Authorization' => 'Basic ' . $this->getAccessToken(),
+                'Accept' => 'application/vnd.Creative Force.v2.1+json',
+                'x-api-key' => $this->apiKey,
             ],
+            'http_errors' => false,
         ]);
     }
 
     /**
-     * Logs any potential errors, clears the API access token from the database
-     * and displays an error message to the user.
+     * Logs any potential errors, clears the API access token from the database and displays an error message to
+     * the user.
      *
      * @param Exception $e
      */
-    private function handleException(Exception $e)
+    public function handleException(Exception $e)
     {
-        variable_del('sso_awardforce_access_token');
+        variable_del('award-force-access-token');
```

Comment:
This looks out of place since the place where the original var was set is now gone.